### PR TITLE
Added sjob page under Running Jobs

### PR DESCRIPTION
**Pull request by Akshit-H-Jain**
Created introduction page for sjob.
Added the page under Running Jobs section.
Updated the index.md file and added sjob page to show on the grid.

### CODE REVIEW

File: docs/source/runningjobs/runningsjob.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+To launch an interactive session, use the following command:
+
+:::{code} bash
+sjob
+:::
+
+This command will display a menu option with different modes of operation.
+
+::::{dropdown} Simple Example of the Menu Options
```
Comment (SerenaCaplins) [inline docs/source/runningjobs/runningsjob.md:26]: Let's make this not a dropdown. The information is concise and may be all many users will need to see.

File: docs/source/runningjobs/runningsjob.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+| **Default**  | To quickly start a job with the default settings.         |
+| **Interactive** | To submit the job interactively with customized resource requests. |
+| **Batch**    | To generate and save `sbatch` scripts with specified parameters. |
+
+## Getting Started with sjob Command
+
+To launch an interactive session, use the following command:
+
+:::{code} bash
+sjob
```
Comment (SerenaCaplins) [inline docs/source/runningjobs/runningsjob.md:21]: We will just need to add the "module load modulename" command here. Lets get it in a module this week.

**Review by SerenaCaplins — COMMENTED** [review]
This all looks great! Only two comments. We can approve this and merge to the main branch once we get the sjobs module built.

File: docs/source/runningjobs/runningsjob.md (hunk 2 above)
Comment (Akshit-H-Jain) [reply docs/source/runningjobs/runningsjob.md]: Added command to load the module.

File: docs/source/runningjobs/runningsjob.md (hunk 1 above)
Comment (Akshit-H-Jain) [reply docs/source/runningjobs/runningsjob.md]: Removed drop drown for user visibility.

File: docs/source/runningjobs/index.md [hunk 3]
```diff
@@ -44,6 +45,16 @@ Run jobs interactively or submit batch scripts for later execution.
 [Learn more »](interactiveandbatch)
 :::
 
+:::{grid-item-card} {octicon}`cpu;1.5em;screen-full` Running Jobs with sjob
+:link: runningsjob
+:link-type: doc
+
+SLURM job submissions with sjob command.
```
Comment (SerenaCaplins) [inline docs/source/runningjobs/index.md:52]: please change this to say job-assist

**Review by SerenaCaplins — APPROVED** [review]
Everything looks great! One change needs to be made to the index card. Otherwise it is good to go public. Thank you.

**Pull request merged**